app/controllers/exam_templates_controller.rb

```diff
@@ -329,4 +329,20 @@ def exam_template_crop_fields_params
             :crop_height
           )
   end
+
+  def list
```

Comment:
Don't make a separate route; instead, pass the data you need in an `exam_templates` field in the existing groups method. This will also save you from having to make two AJAX requests in the component.